suite/performance/flux.go

```diff
@@ -412,14 +441,28 @@ func (f *FluxTest) calculateLatencies(b ginkgo.Benchmarker) error {
 		for contract, contractResults := range testResults {
 			for node, nodeResults := range contractResults {
 				if nodeResults.StartTime.IsZero() {
-					log.Info().Int64("Round ID", roundID).Str("Contract", contract.Address()).Str("Node", node.URL()).Msg("Start time zero")
+					log.Info().
```

Comment:
I put these log lines in for debugging, if keeping probably makes sense to change to Warn level.